src/datachain/lib/dc.py

```diff
@@ -1441,22 +1441,23 @@ def to_parquet(
         )
 
     @classmethod
-    def create_empty(
+    def from_records(
```

Comment:
The problem with `create_empty()` is that it cannot support arbitrary record dictionaries. (See the hardcoded `_datachain_column_types` columns usage below).

We need to improve this API to detect schema from the records (and/or additionally take `schema` parameter for users to pass) before calling it `from_records` (although I don't have strong opinion towards renaming).


Sorry, I should have written something in the issue. We should at least re-open the issue for that to be worked on.